Ludown/lib/parseFileContents.js

```diff
@@ -530,10 +505,11 @@ const parseAndHandleQnAAlterations = function(parsedContent, chunkSplitByLine) {
  * Helper function to parse and handle list entities
  * @param {parserObj} parsedContent parserObj containing current parsed content
  * @param {Array} chunkSplitByLine Array of text lines in the current parsed section
+ * @param {string []} entityRoles Array of possible roles for this entity.
  * @returns {void} Nothing
  * @throws {exception} Throws on errors. exception object includes errCode and text. 
  */
-const parseAndHandleListEntity = function(parsedContent, chunkSplitByLine) {
+const parseAndHandleListEntity = function(parsedContent, chunkSplitByLine, entityRoles) {
     let entityDef = chunkSplitByLine[0].replace(PARSERCONSTS.ENTITY, '').split(':');
```

Comment:
This can be simplified using an Array destructuing assignment:
```js
const [entityName, entityType] = chunkSplitByLine[0].replace(PARSERCONSTS.ENTITY, '').split(':');
```